src/main/java/com/javadiscord/javabot/commands/user_commands/Regex.java

```diff
@@ -0,0 +1,42 @@
+package com.javadiscord.javabot.commands.user_commands;
+
+import com.javadiscord.javabot.commands.ResponseException;
+import com.javadiscord.javabot.commands.SlashCommandHandler;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.requests.restaction.interactions.ReplyAction;
+
+import java.util.regex.Pattern;
+
+public class Regex implements SlashCommandHandler {
+
+    @Override
+    public ReplyAction handle(SlashCommandEvent event) throws ResponseException {
+
+        Pattern pattern = Pattern.compile(event.getOption("regex").getAsString());
+        String string = event.getOption("string").getAsString();
+
+        if (pattern.matcher(string).matches()){
+            return event.replyEmbeds(embedBuilder(true, pattern, string).build());
```

Comment:
you could simplifiy this by doing 
```java
return event.replyEmbeds(embedBuilder(pattern.matcher(string).matches(), pattern, string).build());
```

Reply:
Yup, already did that in a8f411e6364c7d39f582a05c35a4b2ea0b8f68d8